Master

ejimenezramos opened the pull request.

ejimenezramos committed:
170505 arreglo login y register

PabloBarrantes committed:
Add files via upload

"Pablo: he corregido un fallo del diseño en la vista móvil con los textos que acompañan a la imágenes"

ejimenezramos committed:
170505 edición imágenes Elena

ejimenezramos committed:
170508 Cambios register Elena

ejimenezramos committed:
170508 cambios register

ejimenezramos committed:
170509 ele

ejimenezramos committed:
contac test

ejimenezramos committed:
Merge remote-tracking branch 'origin/master'

mohalehhit committed:
moha

mohalehhit committed:
moha

mohalehhit committed:
moha

mohalehhit committed:
Moha

workspace

ejimenezramos committed:
commit bueno

ejimenezramos committed:
Cambios Elena en Registro Jquery y registro vista

PabloBarrantes committed:
Add files via upload

"Pablo: cambios en el main del body: nuevo div para los pack"

PabloBarrantes committed:
pablo cambio register campo username

mohalehhit committed:
Moha Prueba para pablo

PabloBarrantes committed:
Pablo: Eliminación de imagenes innecesarias

ejimenezramos committed:
Elena cambios en register js validación y html

ejimenezramos committed:
Elena más cambios y validaciones en registro js

PabloBarrantes committed:
Pablo: Slider con CSS3 y HTML5

He añadido una clase nueva slider.css, he añadido las 3 imagenes que
forman el slider y en home.php, he quitado el div que contenía el gif y
lo he cambiado por el div que contiene el slider

ejimenezramos committed:
Registro primer filtro commit Elena

ejimenezramos committed:
Cambios validaciones y nuevas páginas Elena

ejimenezramos committed:
ultimo validaciones

ejimenezramos committed:
habilitar boton registro cuando este clickeado el check

ejimenezramos committed:
Cambios en register y validaciones

Actualización en la validación de tarjeta de residencia y dni,  añado
campos de dirección para el registro del cliente y valores a la lista de
provincias

PabloBarrantes committed:
Pablo: primer cambio del home

PabloBarrantes committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	.idea/workspace.xml

ejimenezramos committed:
cambios peticiones AJAX

para añadir el tema de las provincias a las peticiones AJAX cambio el
nombre del archivo para que sea más general y añado los script a las
vistas

ejimenezramos committed:
modifico los ID's correctamente para controlarlos en las peticiones AJAX

PabloBarrantes committed:
Pablo: correcciones del home

ejimenezramos committed:
peticiones AJAX

PabloBarrantes committed:
Pablo: Reajustes home

He vuelto a cambiar la posicion de los titulos y su color. También he
cambiado la foto del medio

PabloBarrantes committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	.idea/workspace.xml

mohalehhit committed:
Productos Moha

ejimenezramos committed:
elenaintento

mohalehhit committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	.idea/workspace.xml

ejimenezramos committed:
pruebacommit

ejimenezramos committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	.idea/workspace.xml

ejimenezramos committed:
intentemosdenuevo

ejimenezramos committed:
intento1

ejimenezramos committed:
cambio color casita en formulario registro y vuelvo a hacer peticiones.php

ejimenezramos committed:
arreglo de peticiones

ejimenezramos committed:
validacion del campo dirección y cp

PabloBarrantes committed:
Pablo: pequeño cambio en diseño packs

ejimenezramos committed:
nuevos cambios en validaciones para el submit

PabloBarrantes committed:
Pablo: Cambios en el home

ejimenezramos committed:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

PabloBarrantes committed:
Pablo: nuevos ajustes en el home

ejimenezramos committed:
CAMBIO VALIDACIONES

ejimenezramos committed:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

PabloBarrantes committed:
Pablo:pequeños cambios en pagina packs

PabloBarrantes committed:
Pablo: añado footer en home

PabloBarrantes committed:
Pablo:reajuste  js de slider

PabloBarrantes committed:
Pablo: cambios en el home, footer, progressbar y pakcs

ejimenezramos committed:
ELENA CAMBIOS EN FECHA Y DIRECCION EN REGISTRO

ejimenezramos committed:
commit prueba

PabloBarrantes commented:
Buenos días, esto que es? XD

2017-05-21 20:48 GMT+02:00 ejimenezramos <notifications@github.com>:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/ejimenezramos/proyectoFinal/pull/1
> Commit Summary
>
>    - 170505 arreglo login y register
>    - Add files via upload
>    - 170505 edición imágenes Elena
>    - 170508 Cambios register Elena
>    - 170508 cambios register
>    - 170509 ele
>    - contac test
>    - Merge remote-tracking branch 'origin/master'
>    - moha
>    - moha
>    - moha
>    - Moha
>    - commit bueno
>    - Cambios Elena en Registro Jquery y registro vista
>    - Add files via upload
>    - pablo cambio register campo username
>    - Moha Prueba para pablo
>    - Pablo: Eliminación de imagenes innecesarias
>    - Elena cambios en register js validación y html
>    - Elena más cambios y validaciones en registro js
>    - Pablo: Slider con CSS3 y HTML5
>    - Registro primer filtro commit Elena
>    - Cambios validaciones y nuevas páginas Elena
>    - ultimo validaciones
>    - habilitar boton registro cuando este clickeado el check
>    - Cambios en register y validaciones
>    - Pablo: primer cambio del home
>    - Merge remote-tracking branch 'origin/master'
>    - cambios peticiones AJAX
>    - modifico los ID's correctamente para controlarlos en las peticiones
>    AJAX
>    - Pablo: correcciones del home
>    - peticiones AJAX
>    - Pablo: Reajustes home
>    - Merge remote-tracking branch 'origin/master'
>    - Productos Moha
>    - elenaintento
>    - Merge remote-tracking branch 'origin/master'
>    - pruebacommit
>    - Merge remote-tracking branch 'origin/master'
>    - intentemosdenuevo
>    - intento1
>    - cambio color casita en formulario registro y vuelvo a hacer
>    peticiones.php
>    - arreglo de peticiones
>    - validacion del campo dirección y cp
>    - Pablo: pequeño cambio en diseño packs
>    - nuevos cambios en validaciones para el submit
>    - Pablo: Cambios en el home
>    - Merge branch 'master' of https://github.com/
>    ejimenezramos/proyectoFinal
>    - Pablo: nuevos ajustes en el home
>    - CAMBIO VALIDACIONES
>    - Merge branch 'master' of https://github.com/
>    ejimenezramos/proyectoFinal
>    - Pablo:pequeños cambios en pagina packs
>    - Pablo: añado footer en home
>    - Pablo:reajuste js de slider
>    - Pablo: cambios en el home, footer, progressbar y pakcs
>
> File Changes
>
>    - *A* .gitignore
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-0>
>    (5)
>    - *A* .idea/encodings.xml
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-1>
>    (6)
>    - *A* .idea/vcs.xml
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-2>
>    (6)
>    - *M* .idea/workspace.xml
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-3>
>    (507)
>    - *M* css/bootstrap.css
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-4>
>    (25)
>    - *M* css/font-awesome.css
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-5>
>    (11)
>    - *A* css/footer.css
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-6>
>    (166)
>    - *M* css/style.css
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-7>
>    (127)
>    - *A* css/w3c.css
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-8>
>    (227)
>    - *D* images/ab.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-9>
>    (0)
>    - *D* images/ab1.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-10>
>    (0)
>    - *D* images/blue.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-11>
>    (0)
>    - *A* images/bonsdes.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-12>
>    (0)
>    - *A* images/boxcheerios.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-13>
>    (0)
>    - *A* images/boxcinnamon.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-14>
>    (0)
>    - *A* images/boxcocoa.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-15>
>    (0)
>    - *A* images/boxfroot.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-16>
>    (0)
>    - *A* images/boxlucky.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-17>
>    (0)
>    - *A* images/boxreeses.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-18>
>    (0)
>    - *A* images/boxtrix.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-19>
>    (0)
>    - *D* images/cac.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-20>
>    (0)
>    - *A* images/cereal1.jpeg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-21>
>    (0)
>    - *M* images/cereal1.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-22>
>    (0)
>    - *A* images/cereal2.jpeg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-23>
>    (0)
>    - *A* images/cereal2.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-24>
>    (0)
>    - *A* images/cereal3.jpeg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-25>
>    (0)
>    - *A* images/cereal3.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-26>
>    (0)
>    - *A* images/chipsdes.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-27>
>    (0)
>    - *D* images/co.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-28>
>    (0)
>    - *D* images/co1.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-29>
>    (0)
>    - *A* images/conguitosdes.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-30>
>    (0)
>    - *A* images/conocenos_home.jpeg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-31>
>    (0)
>    - *D* images/download.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-32>
>    (0)
>    - *D* images/frootloops.jpeg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-33>
>    (0)
>    - *D* images/frootloops.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-34>
>    (0)
>    - *D* images/green.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-35>
>    (0)
>    - *A* images/home_variedad.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-36>
>    (0)
>    - *D* images/ki.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-37>
>    (0)
>    - *D* images/ki1.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-38>
>    (0)
>    - *D* images/ki2.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-39>
>    (0)
>    - *D* images/ki3.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-40>
>    (0)
>    - *D* images/ki4.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-41>
>    (0)
>    - *D* images/ki5.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-42>
>    (0)
>    - *D* images/ki6.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-43>
>    (0)
>    - *D* images/ki7.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-44>
>    (0)
>    - *D* images/ki8.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-45>
>    (0)
>    - *A* images/kinderdes.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-46>
>    (0)
>    - *D* images/leche.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-47>
>    (0)
>    - *D* images/lucky.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-48>
>    (0)
>    - *A* images/mas.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-49>
>    (0)
>    - *D* images/me.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-50>
>    (0)
>    - *D* images/me1.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-51>
>    (0)
>    - *D* images/me2.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-52>
>    (0)
>    - *A* images/milkblue.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-53>
>    (0)
>    - *A* images/milkgreen.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-54>
>    (0)
>    - *A* images/milkpink.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-55>
>    (0)
>    - *A* images/milksojaalmond.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-56>
>    (0)
>    - *A* images/milkvainilla.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-57>
>    (0)
>    - *D* images/of.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-58>
>    (0)
>    - *D* images/of1.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-59>
>    (0)
>    - *D* images/of10.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-60>
>    (0)
>    - *D* images/of11.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-61>
>    (0)
>    - *D* images/of12.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-62>
>    (0)
>    - *D* images/of13.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-63>
>    (0)
>    - *D* images/of14.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-64>
>    (0)
>    - *D* images/of15.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-65>
>    (0)
>    - *D* images/of16.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-66>
>    (0)
>    - *D* images/of17.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-67>
>    (0)
>    - *D* images/of18.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-68>
>    (0)
>    - *D* images/of19.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-69>
>    (0)
>    - *D* images/of2.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-70>
>    (0)
>    - *D* images/of20.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-71>
>    (0)
>    - *D* images/of21.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-72>
>    (0)
>    - *D* images/of22.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-73>
>    (0)
>    - *D* images/of23.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-74>
>    (0)
>    - *D* images/of24.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-75>
>    (0)
>    - *D* images/of25.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-76>
>    (0)
>    - *D* images/of26.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-77>
>    (0)
>    - *D* images/of27.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-78>
>    (0)
>    - *D* images/of28.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-79>
>    (0)
>    - *D* images/of29.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-80>
>    (0)
>    - *D* images/of3.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-81>
>    (0)
>    - *D* images/of30.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-82>
>    (0)
>    - *D* images/of31.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-83>
>    (0)
>    - *D* images/of32.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-84>
>    (0)
>    - *D* images/of33.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-85>
>    (0)
>    - *D* images/of34.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-86>
>    (0)
>    - *D* images/of35.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-87>
>    (0)
>    - *D* images/of36.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-88>
>    (0)
>    - *D* images/of37.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-89>
>    (0)
>    - *D* images/of38.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-90>
>    (0)
>    - *D* images/of39.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-91>
>    (0)
>    - *D* images/of4.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-92>
>    (0)
>    - *D* images/of40.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-93>
>    (0)
>    - *D* images/of41.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-94>
>    (0)
>    - *D* images/of42.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-95>
>    (0)
>    - *D* images/of43.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-96>
>    (0)
>    - *D* images/of44.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-97>
>    (0)
>    - *D* images/of45.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-98>
>    (0)
>    - *D* images/of46.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-99>
>    (0)
>    - *D* images/of47.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-100>
>    (0)
>    - *D* images/of48.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-101>
>    (0)
>    - *D* images/of49.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-102>
>    (0)
>    - *D* images/of5.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-103>
>    (0)
>    - *D* images/of50.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-104>
>    (0)
>    - *D* images/of51.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-105>
>    (0)
>    - *D* images/of52.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-106>
>    (0)
>    - *D* images/of53.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-107>
>    (0)
>    - *D* images/of54.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-108>
>    (0)
>    - *D* images/of55.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-109>
>    (0)
>    - *D* images/of56.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-110>
>    (0)
>    - *D* images/of57.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-111>
>    (0)
>    - *D* images/of58.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-112>
>    (0)
>    - *D* images/of59.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-113>
>    (0)
>    - *D* images/of6.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-114>
>    (0)
>    - *D* images/of7.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-115>
>    (0)
>    - *D* images/of8.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-116>
>    (0)
>    - *D* images/of9.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-117>
>    (0)
>    - *A* images/pack_home.jpeg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-118>
>    (0)
>    - *A* images/packbons.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-119>
>    (0)
>    - *A* images/packchips.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-120>
>    (0)
>    - *A* images/packconguitos.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-121>
>    (0)
>    - *A* images/packkinder.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-122>
>    (0)
>    - *A* images/packoreo.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-123>
>    (0)
>    - *D* images/pink.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-124>
>    (0)
>    - *D* images/si-1.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-125>
>    (0)
>    - *D* images/si.jpg
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-126>
>    (0)
>    - *A* images/toparce.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-127>
>    (0)
>    - *A* images/topcaramelo.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-128>
>    (0)
>    - *A* images/topchocolate.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-129>
>    (0)
>    - *A* images/topfresa.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-130>
>    (0)
>    - *D* images/vainilla.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-131>
>    (0)
>    - *D* images/wi.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-132>
>    (0)
>    - *D* images/wi1.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-133>
>    (0)
>    - *D* images/wi2.png
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-134>
>    (0)
>    - *D* js/contacto.js
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-135>
>    (20)
>    - *A* js/jquery.peticionesAJAX.js
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-136>
>    (37)
>    - *D* js/jquery.productos.js
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-137>
>    (18)
>    - *D* js/productos.js
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-138>
>    (18)
>    - *A* js/validaciones.js
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-139>
>    (331)
>    - *M* mvc/modelo/form/Contact/mdlContact.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-140>
>    (2)
>    - *R* mvc/modelo/form/Home/mdlHome.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-141>
>    (0)
>    - *R* mvc/modelo/form/Olvido/mdlolvido.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-142>
>    (8)
>    - *A* mvc/modelo/form/Terminos/mdlterminos.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-143>
>    (23)
>    - *M* mvc/vista/aboutus.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-144>
>    (103)
>    - *M* mvc/vista/contact.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-145>
>    (204)
>    - *M* mvc/vista/home.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-146>
>    (1297)
>    - *M* mvc/vista/login.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-147>
>    (127)
>    - *D* mvc/vista/menu.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-148>
>    (1363)
>    - *A* mvc/vista/olvido.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-149>
>    (285)
>    - *M* mvc/vista/packs.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-150>
>    (1126)
>    - *M* mvc/vista/productos.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-151>
>    (996)
>    - *M* mvc/vista/register.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-152>
>    (620)
>    - *A* mvc/vista/terminos.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-153>
>    (1071)
>    - *D* requests/compruebaProductos.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-154>
>    (31)
>    - *A* requests/peticionesFiltros.php
>    <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-155>
>    (68)
>
> Patch Links:
>
>    - https://github.com/ejimenezramos/proyectoFinal/pull/1.patch
>    - https://github.com/ejimenezramos/proyectoFinal/pull/1.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ejimenezramos/proyectoFinal/pull/1>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/AYWAl7djq8qG-fSll7fD_F5UMcPqrFSjks5r8IbmgaJpZM4NhrzV>
> .
>

ejimenezramos commented:
eso es la liada de ayer😂😂😂 al hacer mi commit. Porque supongo q entro en conflicto nuevamente, porque no cai en q tu tb habias tocado la página de registro para poner el progress bar, pero al final hice un ultimo commit y estaba todo lo tuyo y lo mio, voy a ver si veo como se llamaba, y ahora te digo por el hangouts

________________________________
De: PabloBarrantes <notifications@github.com>
Enviado: lunes, 22 de mayo de 2017 8:32:37
Para: ejimenezramos/proyectoFinal
Cc: ejimenezramos; Author
Asunto: Re: [ejimenezramos/proyectoFinal] Master (#1)

Buenos días, esto que es? XD

2017-05-21 20:48 GMT+02:00 ejimenezramos <notifications@github.com>:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/ejimenezramos/proyectoFinal/pull/1
> Commit Summary
>
> - 170505 arreglo login y register
> - Add files via upload
> - 170505 edición imágenes Elena
> - 170508 Cambios register Elena
> - 170508 cambios register
> - 170509 ele
> - contac test
> - Merge remote-tracking branch 'origin/master'
> - moha
> - moha
> - moha
> - Moha
> - commit bueno
> - Cambios Elena en Registro Jquery y registro vista
> - Add files via upload
> - pablo cambio register campo username
> - Moha Prueba para pablo
> - Pablo: Eliminación de imagenes innecesarias
> - Elena cambios en register js validación y html
> - Elena más cambios y validaciones en registro js
> - Pablo: Slider con CSS3 y HTML5
> - Registro primer filtro commit Elena
> - Cambios validaciones y nuevas páginas Elena
> - ultimo validaciones
> - habilitar boton registro cuando este clickeado el check
> - Cambios en register y validaciones
> - Pablo: primer cambio del home
> - Merge remote-tracking branch 'origin/master'
> - cambios peticiones AJAX
> - modifico los ID's correctamente para controlarlos en las peticiones
> AJAX
> - Pablo: correcciones del home
> - peticiones AJAX
> - Pablo: Reajustes home
> - Merge remote-tracking branch 'origin/master'
> - Productos Moha
> - elenaintento
> - Merge remote-tracking branch 'origin/master'
> - pruebacommit
> - Merge remote-tracking branch 'origin/master'
> - intentemosdenuevo
> - intento1
> - cambio color casita en formulario registro y vuelvo a hacer
> peticiones.php
> - arreglo de peticiones
> - validacion del campo dirección y cp
> - Pablo: pequeño cambio en diseño packs
> - nuevos cambios en validaciones para el submit
> - Pablo: Cambios en el home
> - Merge branch 'master' of https://github.com/
> ejimenezramos/proyectoFinal
> - Pablo: nuevos ajustes en el home
> - CAMBIO VALIDACIONES
> - Merge branch 'master' of https://github.com/
> ejimenezramos/proyectoFinal
> - Pablo:pequeños cambios en pagina packs
> - Pablo: añado footer en home
> - Pablo:reajuste js de slider
> - Pablo: cambios en el home, footer, progressbar y pakcs
>
> File Changes
>
> - *A* .gitignore
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-0>
> (5)
> - *A* .idea/encodings.xml
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-1>
> (6)
> - *A* .idea/vcs.xml
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-2>
> (6)
> - *M* .idea/workspace.xml
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-3>
> (507)
> - *M* css/bootstrap.css
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-4>
> (25)
> - *M* css/font-awesome.css
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-5>
> (11)
> - *A* css/footer.css
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-6>
> (166)
> - *M* css/style.css
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-7>
> (127)
> - *A* css/w3c.css
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-8>
> (227)
> - *D* images/ab.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-9>
> (0)
> - *D* images/ab1.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-10>
> (0)
> - *D* images/blue.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-11>
> (0)
> - *A* images/bonsdes.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-12>
> (0)
> - *A* images/boxcheerios.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-13>
> (0)
> - *A* images/boxcinnamon.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-14>
> (0)
> - *A* images/boxcocoa.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-15>
> (0)
> - *A* images/boxfroot.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-16>
> (0)
> - *A* images/boxlucky.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-17>
> (0)
> - *A* images/boxreeses.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-18>
> (0)
> - *A* images/boxtrix.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-19>
> (0)
> - *D* images/cac.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-20>
> (0)
> - *A* images/cereal1.jpeg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-21>
> (0)
> - *M* images/cereal1.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-22>
> (0)
> - *A* images/cereal2.jpeg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-23>
> (0)
> - *A* images/cereal2.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-24>
> (0)
> - *A* images/cereal3.jpeg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-25>
> (0)
> - *A* images/cereal3.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-26>
> (0)
> - *A* images/chipsdes.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-27>
> (0)
> - *D* images/co.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-28>
> (0)
> - *D* images/co1.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-29>
> (0)
> - *A* images/conguitosdes.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-30>
> (0)
> - *A* images/conocenos_home.jpeg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-31>
> (0)
> - *D* images/download.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-32>
> (0)
> - *D* images/frootloops.jpeg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-33>
> (0)
> - *D* images/frootloops.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-34>
> (0)
> - *D* images/green.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-35>
> (0)
> - *A* images/home_variedad.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-36>
> (0)
> - *D* images/ki.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-37>
> (0)
> - *D* images/ki1.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-38>
> (0)
> - *D* images/ki2.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-39>
> (0)
> - *D* images/ki3.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-40>
> (0)
> - *D* images/ki4.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-41>
> (0)
> - *D* images/ki5.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-42>
> (0)
> - *D* images/ki6.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-43>
> (0)
> - *D* images/ki7.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-44>
> (0)
> - *D* images/ki8.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-45>
> (0)
> - *A* images/kinderdes.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-46>
> (0)
> - *D* images/leche.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-47>
> (0)
> - *D* images/lucky.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-48>
> (0)
> - *A* images/mas.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-49>
> (0)
> - *D* images/me.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-50>
> (0)
> - *D* images/me1.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-51>
> (0)
> - *D* images/me2.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-52>
> (0)
> - *A* images/milkblue.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-53>
> (0)
> - *A* images/milkgreen.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-54>
> (0)
> - *A* images/milkpink.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-55>
> (0)
> - *A* images/milksojaalmond.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-56>
> (0)
> - *A* images/milkvainilla.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-57>
> (0)
> - *D* images/of.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-58>
> (0)
> - *D* images/of1.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-59>
> (0)
> - *D* images/of10.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-60>
> (0)
> - *D* images/of11.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-61>
> (0)
> - *D* images/of12.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-62>
> (0)
> - *D* images/of13.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-63>
> (0)
> - *D* images/of14.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-64>
> (0)
> - *D* images/of15.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-65>
> (0)
> - *D* images/of16.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-66>
> (0)
> - *D* images/of17.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-67>
> (0)
> - *D* images/of18.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-68>
> (0)
> - *D* images/of19.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-69>
> (0)
> - *D* images/of2.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-70>
> (0)
> - *D* images/of20.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-71>
> (0)
> - *D* images/of21.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-72>
> (0)
> - *D* images/of22.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-73>
> (0)
> - *D* images/of23.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-74>
> (0)
> - *D* images/of24.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-75>
> (0)
> - *D* images/of25.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-76>
> (0)
> - *D* images/of26.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-77>
> (0)
> - *D* images/of27.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-78>
> (0)
> - *D* images/of28.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-79>
> (0)
> - *D* images/of29.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-80>
> (0)
> - *D* images/of3.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-81>
> (0)
> - *D* images/of30.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-82>
> (0)
> - *D* images/of31.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-83>
> (0)
> - *D* images/of32.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-84>
> (0)
> - *D* images/of33.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-85>
> (0)
> - *D* images/of34.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-86>
> (0)
> - *D* images/of35.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-87>
> (0)
> - *D* images/of36.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-88>
> (0)
> - *D* images/of37.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-89>
> (0)
> - *D* images/of38.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-90>
> (0)
> - *D* images/of39.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-91>
> (0)
> - *D* images/of4.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-92>
> (0)
> - *D* images/of40.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-93>
> (0)
> - *D* images/of41.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-94>
> (0)
> - *D* images/of42.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-95>
> (0)
> - *D* images/of43.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-96>
> (0)
> - *D* images/of44.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-97>
> (0)
> - *D* images/of45.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-98>
> (0)
> - *D* images/of46.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-99>
> (0)
> - *D* images/of47.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-100>
> (0)
> - *D* images/of48.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-101>
> (0)
> - *D* images/of49.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-102>
> (0)
> - *D* images/of5.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-103>
> (0)
> - *D* images/of50.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-104>
> (0)
> - *D* images/of51.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-105>
> (0)
> - *D* images/of52.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-106>
> (0)
> - *D* images/of53.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-107>
> (0)
> - *D* images/of54.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-108>
> (0)
> - *D* images/of55.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-109>
> (0)
> - *D* images/of56.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-110>
> (0)
> - *D* images/of57.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-111>
> (0)
> - *D* images/of58.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-112>
> (0)
> - *D* images/of59.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-113>
> (0)
> - *D* images/of6.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-114>
> (0)
> - *D* images/of7.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-115>
> (0)
> - *D* images/of8.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-116>
> (0)
> - *D* images/of9.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-117>
> (0)
> - *A* images/pack_home.jpeg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-118>
> (0)
> - *A* images/packbons.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-119>
> (0)
> - *A* images/packchips.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-120>
> (0)
> - *A* images/packconguitos.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-121>
> (0)
> - *A* images/packkinder.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-122>
> (0)
> - *A* images/packoreo.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-123>
> (0)
> - *D* images/pink.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-124>
> (0)
> - *D* images/si-1.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-125>
> (0)
> - *D* images/si.jpg
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-126>
> (0)
> - *A* images/toparce.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-127>
> (0)
> - *A* images/topcaramelo.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-128>
> (0)
> - *A* images/topchocolate.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-129>
> (0)
> - *A* images/topfresa.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-130>
> (0)
> - *D* images/vainilla.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-131>
> (0)
> - *D* images/wi.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-132>
> (0)
> - *D* images/wi1.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-133>
> (0)
> - *D* images/wi2.png
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-134>
> (0)
> - *D* js/contacto.js
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-135>
> (20)
> - *A* js/jquery.peticionesAJAX.js
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-136>
> (37)
> - *D* js/jquery.productos.js
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-137>
> (18)
> - *D* js/productos.js
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-138>
> (18)
> - *A* js/validaciones.js
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-139>
> (331)
> - *M* mvc/modelo/form/Contact/mdlContact.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-140>
> (2)
> - *R* mvc/modelo/form/Home/mdlHome.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-141>
> (0)
> - *R* mvc/modelo/form/Olvido/mdlolvido.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-142>
> (8)
> - *A* mvc/modelo/form/Terminos/mdlterminos.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-143>
> (23)
> - *M* mvc/vista/aboutus.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-144>
> (103)
> - *M* mvc/vista/contact.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-145>
> (204)
> - *M* mvc/vista/home.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-146>
> (1297)
> - *M* mvc/vista/login.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-147>
> (127)
> - *D* mvc/vista/menu.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-148>
> (1363)
> - *A* mvc/vista/olvido.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-149>
> (285)
> - *M* mvc/vista/packs.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-150>
> (1126)
> - *M* mvc/vista/productos.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-151>
> (996)
> - *M* mvc/vista/register.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-152>
> (620)
> - *A* mvc/vista/terminos.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-153>
> (1071)
> - *D* requests/compruebaProductos.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-154>
> (31)
> - *A* requests/peticionesFiltros.php
> <https://github.com/ejimenezramos/proyectoFinal/pull/1/files#diff-155>
> (68)
>
> Patch Links:
>
> - https://github.com/ejimenezramos/proyectoFinal/pull/1.patch
> - https://github.com/ejimenezramos/proyectoFinal/pull/1.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ejimenezramos/proyectoFinal/pull/1>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/AYWAl7djq8qG-fSll7fD_F5UMcPqrFSjks5r8IbmgaJpZM4NhrzV>
> .
>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ejimenezramos/proyectoFinal/pull/1#issuecomment-303010810>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AZdNDJp8W7wWxiqAr4MvEP_hRzihkNcDks5r8SwFgaJpZM4NhrzV>.

PabloBarrantes committed:
Pablo: packs, footer, licencia y devluciones

mohalehhit committed:
PhP

ejimenezramos committed:
cambios en registro campo fecha onchange probar y aplicar en el onsubmit

PabloBarrantes committed:
Pablo: cambios en el footer

PabloBarrantes committed:
Pablo: pruebas pagina packs

ejimenezramos committed:
cambiuos en validacion fecha

ejimenezramos committed:
+ cambios en validacion campos del registro

PabloBarrantes committed:
Pablo: cambios pagina packs

mohalehhit committed:
No sé lo que es

mohalehhit committed:
Ni puta idea 2

# Conflicts:
#	.idea/workspace.xml

mohalehhit committed:
veamos

ejimenezramos committed:
cambios en validacion rgistro

PabloBarrantes committed:
Pablo: cambios en footer, home y titulo header

ejimenezramos committed:
comprobacion campos blancos y check en registro a falta de validacion

ejimenezramos committed:
validación parcial con limpieza de campos

ejimenezramos committed:
casi validado vamonoooos que queda poco

ejimenezramos committed:
cambio en getFullyear en funcion comprobar bisiesto, por el año introducido

mohalehhit committed:
Register modelo

mohalehhit committed:
exec usuarios

ejimenezramos committed:
commit elena cambios en val

ejimenezramos committed:
algunos cambios más para firefox

PabloBarrantes committed:
Pablo: pequeños ajustes en diseño

PabloBarrantes committed:
Pablo: implementacion buscador

Para probar el buscador teneis que crear las tablas nuevas de productos
y un insert con los datos
estan en el trello

PabloBarrantes committed:
Pablo: he añadido las licencias

PabloBarrantes committed:
Pablo: commit prueba

PabloBarrantes committed:
Pablo: pequeños ajustes buscador

ejimenezramos committed:
elena subida prefinal registro

ejimenezramos committed:
cambio nombre y quito alerts

PabloBarrantes committed:
Pablo: cambio mensaje buscador

ejimenezramos committed:
cambio nombres archivos

ejimenezramos committed:
nuevo cambio nombre archivos

ejimenezramos committed:
commit

ejimenezramos committed:
ya está arregladoo!!!Vamonoooooos!

PabloBarrantes committed:
Pablo: cambios en los modales

ejimenezramos committed:
subida registro

PabloBarrantes committed:
Pablo: cambio modales en packs y productos

ejimenezramos committed:
prueba

PabloBarrantes committed:
Pablo: nuevo jquery de registro

Commit:
hsdhysdhefsd

PabloBarrantes committed:
Pablo: implementacion Login

Commit:
Prueba

Commit:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

Commit:
Login, Registro, Home y varios cambios en estilos de plantilla original

Commit:
Commit Anterior

Commit:
Funciona, anda

mohalehhit committed:
A ver si va

Commit:
Productos, home y algo más

Commit:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

mohalehhit committed:
Final

mohalehhit committed:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

# Conflicts:
#	.vs/proyectoFinal/v14/.suo
#	mvc/modelo/form/Productos/ProductosParser.php
#	mvc/modelo/form/Productos/mdlProductos.php
#	mvc/vista/licencia.php
#	mvc/vista/productos.php
#	proyectoFinal.phpproj

mohalehhit committed:
Mier

mohalehhit committed:
asd

PabloBarrantes committed:
pablo: correccion fotos

mohalehhit committed:
Cambios en el case login y jquery.registro

mohalehhit committed:
Cosas de chicas

ejimenezramos committed:
cambio color sobre e icono de area

ejimenezramos committed:
registro con filter

ejimenezramos committed:
validacion login

ejimenezramos committed:
validación olvido contraseña

ejimenezramos committed:
subida validación contacto

ejimenezramos committed:
subida rectificado login y arreglo en productos para filtrar las vistas

ejimenezramos committed:
subida nueva propuesta packs para pablo

ejimenezramos committed:
ya saca datos del carro

PabloBarrantes committed:
Pablo: registro ok, vista user

PabloBarrantes committed:
Pablo: pequeños ajustes

PabloBarrantes committed:
Pablo: cambio diseño packs

ejimenezramos committed:
carro funciona y hace submit

ejimenezramos committed:
contenido de aboutus

ejimenezramos committed:
subida de fuente, centro precio productos, contenido de about us

ejimenezramos committed:
cambio en estilos migas de pan

ejimenezramos committed:
cambio de la etiqueta title en cada vista

ejimenezramos committed:
subida imágenes packs

ejimenezramos committed:
cambio fuente titulo

PabloBarrantes committed:
Pablo: cambios en packs

mohalehhit committed:
Multiples cambios que ni recuerdo

ejimenezramos committed:
cambio del campo cp de number a text

PabloBarrantes committed:
Pablo: correccion footer y nuevos estilos boton packs

ejimenezramos committed:
subida con carro desde cliente

ejimenezramos committed:
correccion jquery y cambio nombre productos

PabloBarrantes committed:
Pablo: cambio en sobre nosotros

ejimenezramos committed:
quitamos fecha y password de mi cuenta

mohalehhit committed:
asdasdd

ejimenezramos committed:
igualo contacto a registro para que desaparezcan los errores en onchange

ejimenezramos committed:
cambio de la imagen de packs chocolate al incorporar al carrito

ejimenezramos committed:
cambio titles de todas las páginas

ejimenezramos committed:
arreglo login front y back

ejimenezramos committed:
correcion del case alphanum_space tb en register

ejimenezramos committed:
solcuión en error tipo direccion al no seleccionar nada

ejimenezramos committed:
cambio enlaces del footer redes sociales

ejimenezramos committed:
cambio para que en contacto coja nombre y apellido con acentos

mohalehhit committed:
commit moha

mohalehhit committed:
commit con compra solo logueado

mohalehhit committed:
Cambios Mensaje y carrito

mohalehhit committed:
Varios cambios

mohalehhit committed:
Cambios Varios

Arreglado error DNI en registro, arreglo de summary carrito,
implementación de carrito finalizada con todos los insert y con stock.

Varios cambios más que no recuerdo.

PabloBarrantes committed:
Pablo: cambio footer y validacion CSS

PabloBarrantes committed:
Pablo: arreglo caracteres especiales

PabloBarrantes committed:
arreglo tilde

ejimenezramos committed:
commit peligroso

ejimenezramos committed:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

ejimenezramos committed:
commit peligroso2

ejimenezramos committed:
ultimos

mohalehhit committed:
Ultimisimo

ejimenezramos committed:
arreglo footer y carro en el header

ejimenezramos committed:
Merge branch 'master' of https://github.com/ejimenezramos/proyectoFinal

ejimenezramos committed:
ultimos cambios y ajustes

ejimenezramos committed:
README.md
